src/main/scala/mesosphere/marathon/api/EndpointsHelper.scala

```diff
@@ -1,6 +1,6 @@
 package mesosphere.marathon.api
```

Comment:
we're trying to migrate things to double package blocks when we change them, e.g.:

```
package mesosphere.marathon
package api
```